### PR TITLE
[CWS-695] Remove RAD docs

### PR DESCRIPTION
**Pull request by DominicBurkart**
### What does this PR do? What is the motivation?

The security products are sunsetting public-facing runtime anomaly detection. This PR removes references to runtime anomaly detection in the public docs.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/security/cloud_security_management/setup/csm_enterprise.md [hunk 1]
```diff
@@ -133,7 +120,6 @@ Alternatively, use the following examples to enable CSM Enterprise:
 2. Restart the Agent.
 
 
-[1]: /security/threats/runtime_anomaly_detection
 [2]: https://github.com/DataDog/datadog-operator/blob/main/docs/configuration.v2alpha1.md
```
Comment (diogocp) [inline content/en/security/cloud_security_management/setup/csm_enterprise.md:123]: Renumber?

File: content/en/security/cloud_security_management/setup/csm_workload_security.md [hunk 2]
```diff
@@ -95,7 +83,6 @@ Alternatively, use the following examples to enable CSM Workload Security:
 2. Restart the Agent.
 
 
-[1]: /security/threats/runtime_anomaly_detection
 [2]: https://github.com/DataDog/datadog-operator/blob/main/docs/configuration.v2alpha1.md
```
Comment (diogocp) [inline content/en/security/cloud_security_management/setup/csm_workload_security.md:86]: Renumber?

**Review by diogocp — APPROVED** [review]

File: content/en/security/cloud_security_management/setup/csm_workload_security.md (hunk 2 above)
Comment (DominicBurkart) [reply content/en/security/cloud_security_management/setup/csm_workload_security.md:86]: Screenshot 2023-11-15 at 18 25 17

File: content/en/security/cloud_security_management/setup/csm_enterprise.md (hunk 1 above)
Comment (DominicBurkart) [reply content/en/security/cloud_security_management/setup/csm_enterprise.md:123]: Screenshot 2023-11-15 at 18 25 17

**Pull request merged**